doc/user/user/pipelines/index.md

```diff
@@ -55,7 +55,7 @@ The default cart page will allow you to choose from the pipelines that are curre
 
 ![Select a pipeline page.](images/pipeline-tab.png)
 
-Click on the "Select" button at the bottom, right-hand side of the pipeline that you'd like to launch.
+Click on the "Select" button at the bottom of the pipeline that you'd like to launch.
```

Comment:
Somewhere above there's a missing image.  

`[2020-03-19 13:17:40] ERROR `/user/user/pipelines/images/queue-counts.png' not found.`

Reply:
Added in 57065d13224ec849ba765968bf10659f29217363